skbio/alignment/_tabular_msa.py

```diff
@@ -570,3 +570,48 @@ def reindex(self, key=None, keys=None):
             keys_.flags.writeable = False
 
         self._keys = keys_
+
+    @experimental(as_of='0.4.0-dev')
+    def append(self, sequence):
+        """Append a sequence to the MSA.
+
+        Parameters
+        ----------
+        sequence : IUPACSequence
+            Sequence to be appended. Must match the dtype of the MSA and the
+            length of the second dimension of the MSA.
+
+        Raises
+        ------
+        TypeError
+            If the type of the sequence does not match the dtype of the MSA.
+        ValueError
+            If the length of the sequence does not match the length of the
+            second dimension of the MSA.
+
+        Notes
+        -----
+        This operation, even if successful, is not necessarily meaningful.
```

Comment:
Can you explain why it is not meaningful? We were wanting to make it clear to users that the sequences are not re-aligned to account for the new sequence. This is simply adding a sequence to the MSA that matches its # of positions and sequence type.